src/Apps/Order/Routes/Status/index.tsx

```diff
@@ -146,6 +176,19 @@ const offerMessages = {
       <a href="mailto:orders@artsy.net">orders@artsy.net</a>.
     </>
   ),
+  CANCELED: () => (
+    <>
+      <p>
+        Thank you for your repsonse. The seller will be informed of your
```

Comment:
`response`

Reply:
We actually do have a generic "canceled" status page for offers that's slightly different from this, and actually much more similar to the generic canceled page for orders. (See zpl.io/VqlAOY7)

Since `buyerRejectedOffer` is the only _special_ canceled case we care about, I'm not sure where this logic should live but I think we should make sure to only show this message in that one case, not for all canceled offers.

Reply:
Fixed.